### PR TITLE
Update Writing_Good_Commits.rst

### PR DESCRIPTION
**Pull request by pranav16255**
changed the line "It should describe..."

### CODE REVIEW

**Comment by manu-chroma** [comment]
https://github.com/coala/documentation/pull/190#issuecomment-253770404

**Comment by manu-chroma** [comment]
https://github.com/coala/documentation/pull/190#issuecomment-253770404

File: Developers/Writing_Good_Commits.rst [hunk 1]
```diff
@@ -8,13 +8,13 @@ At coala we are looking heavily at the maintainability of the code.
     Code is more often read than written!
 
 It is obvious that we need good code. In order to do that we are
-verifying that every change to our code (i.e. the commits) is making it
+verifying that every change in our code (i.e. the commits) is making it
```
Comment (adtac) [inline Developers/Writing_Good_Commits.rst]: Umm is this change valid? You "make changes *to* the code*. But I guess `in` sounds alright too. Not sure.

**Comment by adtac** [comment]
Also please ensure you go through every point on gitmate's comment. From what I'm seeing, your commit message isn't up to mark. Thanks!

**Pull request closed without merging**